### ISSUE TITLE
LFS bandwidth leak with new permissions

### ISSUE THREAD

**Issue by DaveKriegerEmerald** [integrations/slack#893]
**tl/dr**: After approving a request for more permissions for the GitHub/Slack integration this morning, LFS bandwidth started to be consumed at a rate of about 1.25GB per minute. This stopped when we uninstalled the integration and resumed when we reinstalled it.

Details: We received an email asking for additional permissions for the integration, with a link to https://github.com/organizations/[organization]/settings/installations/[number]/permissions/update. After following the link and approving the requested additional permissions, we received an email from GitHub warning us that 80% of our (at that time) 50GB of monthly LFS bandwidth had been consumed, and 8 minutes later, another email saying that 100% had been consumed. We purchased additional 50GB data packs and monitored the increase. We eliminated other possible factors before we tried uninstalling the Slack integration and observed that the bandwidth leak stopped. It resumed when we reinstalled it.

**Comment by scepticulous** [comment]
HI @DaveKriegerEmerald,
thanks for bringing this up.
This huge increase in LSF traffic is hard to explain. Especially in the context of the most recent update/permission change for Deployments and Checks.

In my opinion, the best way forward would be creating a 'minimal' example. 
E.g. unsubscribing (`/github unsubscribe org) feature` ) from all features one by one and observing the traffic consumption (if you can do that). This way it would be possible to pinpoint the offending 'feature'. 

Either way, I recommend [getting in touch with the GitHub Support](https://github.com/contact?form%5Bsubject%5D=Re:+GitHub%2BSlack+Integration). They will be able to offer better and further assistance. Especially, because this causes extra costs for you.
When contacting them, details about the private Repo, the slack subscription (org/repo and features) and all the details you can provide will be very helpful for their investigation.

**Comment by DaveKriegerEmerald** [comment]
Thank you for the support link! We will try the differential diagnosis you described also.

**Comment by DaveKriegerEmerald** [comment]
Our investigation is indicating that the correlation with the Slack integration being active is probably coincidental; closing this ticket.
